documentation/working-with-testcafe-studio/recording-tests/test-actions/assertions.md

```diff
@@ -0,0 +1,197 @@
+*Actual*             | Any Type &#124; Selector's Property | A value that should be truthy.
+
+## Not Ok
+
+Asserts that *Actual* is false.
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
+*Actual*             | Any Type &#124; Selector's Property | A value that should be falsy.
+
+## Contains
+
+Asserts that *Actual* contains *Expected*.
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
+*Actual*             | String &#124; Array &#124; Object &#124; Selector's Property | A string that contains the *Expected* substring, an array that contains the *Expected* value or an object that contains the *Expected* property.
+*Expected*             | Any type | The expected value.
+
+## Not Contains
+
+Asserts that *Actual* does not contain *Expected*.
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
+*Actual*             | String &#124; Array &#124; Object &#124; Selector's Property | A string that should not contain the *Expected* substring, an array that should not contain the *Expected* value or an object that should not contain the *Expected* property.
+*Expected*             | Any type | The expected value.
+
+## Type of
+
+Asserts that the *Actual* type corresponds to a particular type.
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
+*Actual*             | Any Type &#124; Selector's Property  | A value to verify.
+*Type Name*             | String | The expected type of the *Actual* value.
+
+## Not Type of
+
+Asserts that the *Actual* type is not correspond to a particular type.
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
+*Actual*             | Any Type &#124; Selector's Property | A value to verify.
+*Type Name*             | String | An unexpected type of the *Actual* value.
```

Comment:
> An unexpected type of the *Actual* value.

Unexpected means sudden first of all.

`A type to which the *Actual* value is not expected to belong.`